test/integration/yank_test.rb

```diff
@@ -67,7 +65,6 @@ class YankTest < SystemTest
   end
 
   test "undo a yank is not supported" do
-    skip "This doesn't actually test undoing yank"
```

Comment:
> This doesn't actually test undoing yank

A bit confused about why it says that. The test correctly verifies that unyanking `0.0.0` doesn't mark it as `indexed`.